egs/multi_conv_zh_es_ta/ST/local/prepare_st_transcripts.py

```
@@ -5,12 +5,13 @@
 This script prepares transcript_words.txt from cutset
 """
 
-from lhotse import CutSet
 import argparse
 import logging
+import os
 import pdb
```

Ruff (0.15.18)

[error] Import for `pdb` found

(T100)